netbox/utilities/templatetags/form_helpers.py

```diff
@@ -32,6 +34,15 @@ def getfield(form, fieldname):
         return None
 
 
+@register.filter()
+def getfilterfield(form, fieldname):
+    field = getfield(form, f'{fieldname}')
+    if field is not None:
+        return field
+    else:
+        return getfield(form, f'{fieldname}_id')
```

Comment:
We can simplify this, but I'm not sure how confident we can be in the logic. An explicit mapping of columns to form fields would be preferable.
```suggestion
    return getfield(form, f'{fieldname}') or getfield(form, f'{fieldname}_id')
```